hrjobcontract/js/src/job-contract/controllers/contract.js

```diff
@@ -134,22 +134,22 @@ define([
       vm.fetchRevisionDetails = function(revision) {
         return $q.all([
           ContractDetailsService.getOne({
-            jobcontract_revision_id: revision.jobcontract_revision_id
+            jobcontract_revision_id: revision.jobcontract_revision_id || revision.id
```

Comment:
`var revisionID = revision.jobcontract_revision_id || revision.id;`
Use `revisionID` in all instances